nifi-extension-bundles/nifi-gcp-bundle/nifi-gcp-services-api/pom.xml

```diff
@@ -31,5 +31,23 @@
             <groupId>com.google.auth</groupId>
             <artifactId>google-auth-library-oauth2-http</artifactId>
         </dependency>
+        <!-- please do not remove grpc-* dependencies because they are used by GCP processors at runtime,
+             see also https://github.com/apache/nifi/pull/8491#issuecomment-1992542785 -->
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-api</artifactId>
```

Comment:
scope for these four should be 'runtime' if they're not necessary for compile but needed for tests/execution and makes their necessity explicit.